### PR TITLE
Adds release announcement for week ending 01/28

### PR DESCRIPTION
**Pull request by JosephSalisbury**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/main/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

**Comment by gianfranco-l** [comment]
nothing from honeybadger

**Review by webwurst — APPROVED** [review]
I think a trumpet emoji is missing for the potential break ;) But fine for me

**Comment by gianfranco-l** [comment]
@JosephSalisbury  is there still time to edit this? we (honebadger) have something to announce this week

**Comment by PrashantR31** [comment]
Nothing from Rainbow for this week.

File: src/content/changes/highlights/2022-01-28.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+* [external-dns-app](https://github.com/giantswarm/external-dns-app/) [v2.9.0](https://github.com/giantswarm/external-dns-app/blob/master/CHANGELOG.md#290---2022-01-24) contains changes to mitigate rate limiting on AWS clusters. Please check the changelog for additional notes if you already specify '--aws-batch-change-interval' or '--aws-zones-cache-duration'.
+- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#280---2022-01-27) contains a potential breaking change, and disables the configuration setting 'use-forwarded-headers' by default, to increase security. Please check the changelog for additional notes if you require this configuration.
```
Comment (rossf7) [inline src/content/changes/highlights/2022-01-28.md:13]:
```suggestion
- [nginx-ingress-controller-app](https://github.com/giantswarm/nginx-ingress-controller-app) [v2.8.0](https://github.com/giantswarm/nginx-ingress-controller-app/blob/master/CHANGELOG.md#280---2022-01-27) contains a potential breaking change, and disables the configuration setting 'use-forwarded-headers' by default, to increase security. Please check the changelog for additional notes if you require this configuration.

## Documentation

- We added documentation for [creating an app catalog](https://docs.giantswarm.io/app-platform/create-catalog/) for deploying apps using Helm charts from the community or publishing your own apps.
```

@JosephSalisbury OK if we add this from honeybadger?

cc @gianfranco-l

Comment (JosephSalisbury) [reply src/content/changes/highlights/2022-01-28.md:13]: @rossf7 whatever you need bud <3

Comment (rossf7) [reply src/content/changes/highlights/2022-01-28.md:13]: Thanks @JosephSalisbury :) I'll get this added then. As we've had this gap in docs for a while.

Comment (gianfranco-l) [reply src/content/changes/highlights/2022-01-28.md:13]: @JosephSalisbury  that's all from honeybadger side :D

**Pull request merged**